src/util/positioning.ts

```diff
@@ -181,6 +181,16 @@ export function positionElements(
     'right-top', 'right-bottom'
   ];
   const classList = targetElement.classList;
+  const addClasses = (primary: string, secondary: string) => {
+    if (baseClass) {
+      classesToAdd.push(`${baseClass}-${primary}`);
```

Comment:
Using `classesToAdd` is super confusing here to me :)

My thought process:
- looking at line 186 → whoa, but where is `classesToAdd` defined ?
- oh, it's much later on line 221... but... it is undefined, how this should work?
- oh it's initialised in the loop on line 225

It should just be a function-local variable with this new implementation, no ?
Otherwise, I'd say, pass it as an argument